content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -94,6 +94,7 @@ For a reference to old feature gates that are removed, please refer to
 | `DownwardAPIHugePages` | `false` | Beta | 1.21 | 1.21 |
 | `DownwardAPIHugePages` | `true` | Beta | 1.22 | |
 | `DynamicResourceAllocation` | `false` | Alpha | 1.26 | |
+| `ElasticIndexedJob` | `true` | Beta` | 1.27 | |
```

Comment:
Please add a short description for this feature at the end of the page, as we did for other features.

Reply:
Done